### PR TITLE
feat: analyzer admin api

### PR DESCRIPTION
**Pull request by theus77**
|Q              |A  |
|---------------|---|
|Bug fix?       |n|
|New feature?   |y|	
|BC breaks?     |n|	
|Deprecations?  |n|	
|Fixed tickets  |n|

### CODE REVIEW

File: src/Core/Mapping/AnalyzerManager.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    public function getEntityName(): string
+    {
+        return 'analyzer';
+    }
+
+    public function getAliasesName(): array
+    {
+        return [
+            'analyzers',
```
Comment (AdrienHq) [inline src/Core/Mapping/AnalyzerManager.php:37]: 'analyzer' is missing ?

File: src/Core/Mapping/AnalyzerManager.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+namespace EMS\CoreBundle\Core\Mapping;
+
+use EMS\CommonBundle\Entity\EntityInterface;
+use EMS\CoreBundle\Entity\Analyzer;
+use EMS\CoreBundle\Repository\AnalyzerRepository;
+use EMS\CoreBundle\Service\EntityServiceInterface;
+
+class AnalyzerManager implements EntityServiceInterface
+{
+    private AnalyzerRepository $AnalyzerRepository;
```
Comment (Davidmattei) [inline src/Core/Mapping/AnalyzerManager.php:12]:
```suggestion
    private AnalyzerRepository $analyzerRepository;
```

Comment (Davidmattei) [reply src/Core/Mapping/AnalyzerManager.php:12]: And change in this class $this->AnalyzerRepositoryto $this->analyzerRepository

**Review by Davidmattei — CHANGES_REQUESTED** [review]

File: src/Core/Mapping/AnalyzerManager.php (hunk 1 above)
Comment (theus77) [reply src/Core/Mapping/AnalyzerManager.php:37]: It's in the getEntityName
```
    public function getEntityName(): string
    {
        return 'analyzer';
    }
```

**Review by Davidmattei — APPROVED** [review]

**Pull request merged**